<?xml version="1.0" encoding="UTF-8"?>
<class xmlns="http://xml.phpdox.net/src" full="spec\PhpSpec\Exception\ExceptionSpec" namespace="spec\PhpSpec\Exception" name="ExceptionSpec" abstract="false" final="false" start="9" end="21">
  <file path="/home/robert/Desktop/Hogeschool/2ejaars Development/DevBOpdrachten/project56/http/laravel/vendor/phpspec/phpspec/spec/PhpSpec/Exception" file="ExceptionSpec.php" realpath="/home/robert/Desktop/Hogeschool/2ejaars Development/DevBOpdrachten/project56/http/laravel/vendor/phpspec/phpspec/spec/PhpSpec/Exception/ExceptionSpec.php" size="400" time="2015-12-17T18:44:10+00:00" unixtime="1450377850" sha1="38cabe440a767d8057aced3ebcff814c9ec027fc" relative="vendor/phpspec/phpspec/spec/PhpSpec/Exception/ExceptionSpec.php"/>
  <extends full="PhpSpec\ObjectBehavior" namespace="PhpSpec" name="ObjectBehavior"/>
  <method name="it_extends_basic_exception" start="11" end="14" abstract="false" final="false" static="false" visibility="public"/>
  <method name="it_could_have_a_cause" start="16" end="20" abstract="false" final="false" static="false" visibility="public">
    <parameter name="cause" byreference="false" type="object">
      <type full="ReflectionMethod" namespace="" name="ReflectionMethod"/>
    </parameter>
  </method>
</class>
